### PR TITLE
Features/275 save button

### PR DESCRIPTION
**Pull request by donaier**
save button does not require 'brick_content'

@ryabrody, @effkay

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -26,6 +26,7 @@
 ActiveRecord::Migration.check_pending! if defined?(ActiveRecord::Migration)
 
 RSpec.configure do |config|
+  config.infer_spec_type_from_file_location!
```
Comment (ryabrody) [inline spec/spec_helper.rb]: @donaier 
for what is this?

**Comment by ryabrody** [comment]
@donaier 
pls add changelog :)

File: spec/spec_helper.rb (hunk 1 above)
Comment (donaier) [inline spec/spec_helper.rb]: so the spec is interpreted as a controller spec (http://stackoverflow.com/a/6296767)

Comment (donaier) [inline spec/spec_helper.rb]: it's the line you need in your feature branch to have running specs ;-)

**Pull request merged**